pom.xml

```diff
@@ -792,8 +764,8 @@
                         <rules>
                             <bannedDependencies>
                                 <excludes>
+                                    <!-- TODO very likely wrong -->
```

Comment:
comment here https://github.com/jenkinsci/blueocean-plugin/pull/2433#discussion_r1187977213

Reply:
Edit the comment if you like, but can we ship this, or perhaps some more minimal version limited to `caffeine-api` without a general BOM cleanup?

Reply:
except the comment. everything looks good.

Reply:
https://github.com/jenkinsci/code-coverage-api-plugin/issues/647